### PR TITLE
adjusting content to fit with official doc published in ODF 4.10 docs

### PR DESCRIPTION
**Pull request by likid0**

### CODE REVIEW

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 1]
```diff
@@ -639,28 +564,63 @@ mon.ceph5 ceph5
...
+mgr.ceph-bos-2.ycgwyz ceph-bos-2
+mgr.ceph-bos-5.kremtt ceph-bos-5
+....
+
+
+== Deploying Ceph services
+
+This section provides examples of how you can deploy the following Ceph services:
+
+**Object Storage Devices (OSDs)
```
Comment (netzzer) [inline training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:609]: Does not look right in Preview version. Is this supposed to be a bullet? Same issue with other ** lines.

Preview version - https://github.com/red-hat-storage/ocs-training/blob/a7e76f8e326e3412845af58d4baa0c47b4bfd371/training/modules/ocs4/pages/rhcs-stretched-deploy.adoc

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 2]
```diff
@@ -753,35 +713,38 @@ ID   CLASS  WEIGHT   TYPE NAME       STATUS  REWEIGHT  PRI-AFF
...
-+---------+------+------+------+----+-----+
+3 rbdpool
+....
+
+6.Using cephadm, deploy two new MDS daemons one in each datacenter. For more
+information, see
+link:https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/5/html/operations_guide/management-of-mds-service-using-the-ceph-orchestrator[Management
+of MDS service using the Ceph Orchestrator.]
+ 
+ In our example, we are using hosts `ceph3` and `ceph6`:
```
Comment (netzzer) [inline training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:743]: `ceph3` and `ceph6` formatting maybe wrong for asciidoc.

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 3]
```diff
@@ -639,28 +564,63 @@ mon.ceph5 ceph5
...
-available drives, each drive it finds that is available to be used by ceph will
-be configured as an osd.
+IMPORTANT: it is mandatory to deploy the OSDs for the cluster to work.
 
-We first do a `--dry-run` to check if we would achieve our desired outcome with
-the current command, when we run `ceph orch apply osd --all-available-devices
---dry-run` command it has to scan the hosts for available disks.
+*Procedure*
 
+1.Scan the hosts to check for available disks and to get the OSD layout. In this example, 6 OSDs will be created one per each ceph hosts where `ceph7` is the arbiter node which does not have any OSD's configured.
```
Comment (netzzer) [inline training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:621]: Should there be more space between 1. and text (same issue with all numbered sentences).

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 4]
```diff
@@ -1214,20 +1118,17 @@ replicated_rule
...
 
 [source,role="execute"]
 ....
 ceph mon enable_stretch_mode ceph7 stretch_rule datacenter
 ....
 
-Verify all our pools are using the `stretch_rule` CRUSH rule we have created in our Ceph cluster:
+In this example, `ceph7` is the arbiter node, `stretch_rule` is the crush rule we created in the previous step and `datacenter` is the dividing bucket.
+
+$ Verify all our pools are using the `stretch_rule` CRUSH rule we have created in our Ceph cluster:
```
Comment (netzzer) [inline training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:1131]: The rbdpool is not in this list.

**Review by netzzer — CHANGES_REQUESTED** [review]

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 5]
```diff
@@ -63,52 +63,66 @@ When the missing data center comes back, the cluster will enter recovery stretch
...
 |===
-|Node name|CPU|Memory|Datacenter|Ceph components
-|ceph1|3|8 GB|DC1| OSD+MON
-|ceph2|3|8 GB|DC1| OSD+MON
-|ceph3|3|8 GB|DC1| OSD+MDS+RGW
-|ceph4|3|8 GB|DC2| OSD+MON
-|ceph5|3|8 GB|DC2| OSD+MON
-|ceph6|3|8 GB|DC2| OSD+MDS+RGW
-|ceph7|3|8 GB|DC3| MON
+|Node name|Datacenter|Ceph components
```
[the remark this answers is not in the document]
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:93]: You are right, fixed in new vesion

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc (hunk 4 above)
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc:1190]: Thanks, fixed

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 6]
```diff
@@ -639,28 +564,63 @@ mon.ceph5 ceph5
...
+mgr.ceph2.kremtt ceph2
+....
+
+You can see that there are 2 managers on nodes datacenter1, and no managers in datacenter2. Refer to the table in the Hardware requirements section to verify the layout of the ceph servers per datacenter.
+
+
+5.Move a manager to the node `ceph5` located in our datacenter2 site.
+
+....
+$ ceph orch apply mgr --placement="ceph-bos-2,ceph-bos-5"
```
[the remark this answers is not in the document]
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Fixed, thanks

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 7]
```diff
@@ -639,28 +564,63 @@ mon.ceph5 ceph5
...
+
+6.Verify that the managers are in the correct layout:
+....
+$ ceph orch ps | grep mgr | awk '{print $1 " " $2}'
+....
+
+.Example output:
+....
+mgr.ceph-bos-2.ycgwyz ceph-bos-2
+mgr.ceph-bos-5.kremtt ceph-bos-5
```
[the remark this answers is not in the document]
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Done

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc [hunk 8]
```diff
@@ -623,7 +548,7 @@ ceph orch apply mon --placement="ceph1,ceph3,ceph4,ceph5,ceph7"
 Scheduled mon update...
 ....
 
-We have to verify that we now have the right placement layout for our monitors:
+3.Verify that the monitors are in the correct layout:
```
[the remark this answers is not in the document]
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Fixed thanks

File: training/modules/ocs4/pages/rhcs-stretched-deploy.adoc (hunk 3 above)
Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Yes, I have fixed the issue using list continuations and open blocks

Comment (netzzer) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Still see this below `Create and enable RBD a new block pool.`. Other numbering syntax fixed.

5.Verify that the RBD pool has been created.

Comment (likid0) [reply training/modules/ocs4/pages/rhcs-stretched-deploy.adoc]: Thanks, fixed now

**Review by netzzer — APPROVED** [review]

**Pull request merged**